politeiawww/api/v1/api.md

```diff
@@ -597,7 +568,12 @@ Allows a user to reset his password without being logged in.
 | verificationtoken | string | The verification token which is sent to the user's email address. | Yes |
 | newpassword | String | The new password for the user. | Yes |
 
-**Results:** none
+**Results:** 
+
+| Parameter | Type | Description |
+|-|-|-|
+| verificationtoken | String | The verification token which is required when calling [Verify update user key](#verify-update-user-key). If an email server is set up, this property will be empty or nonexistent |
```

Comment:
Copy/paste error, this shouldn't be `[Verify update user key](#verify-update-user-key)`. This command is special because it has to be called twice, the 2nd time the caller needs to supply the `verificationtoken`. Can you change it to say that instead?